parkhangs-webapp/src/features/modal/ModalParkDetail.js

```diff
@@ -134,48 +134,48 @@ class ModalParkDetail extends React.Component {
                         <div className="ParkLocationDetails">
                             <div>
-                                Address: {park.streetNumber + " " + park.streetName}
+                                <b>Address:</b> {park.streetNumber + " " + park.streetName}
                             </div>
                             <div>
-                                Lat: {park.googleMapsLatLon[0]}
+                                <b>Lat:</b> {park.googleMapsLatLon[0]}
                             </div>
                             <div>
-                                Lon: {park.googleMapsLatLon[1]}
+                                <b>Lon:</b> {park.googleMapsLatLon[1]}
                             </div>
                         </div>
                         <div className="OtherDetails">
                             <div>
-                                Neighbourhood Name: {park.neighbourhoodName}
+                                <b>Neighbourhood Name:</b> {park.neighbourhoodName}
                             </div>
                             <div>
-                                Neighbourhood website:
+                                <b>Neighbourhood website:</b>
                             </div>
                             <a href={park.neighbourhoodURL}>{park.neighbourhoodURL}</a>
                             <div>
-                                Size (in hectares): {park.hectares}
+                                <b>Size (in hectares): </b>{park.hectares}
                             </div>
                             <div>
-                                Washrooms available?: {park.hasWashrooms ? "yes" : "no"}
+                                <b>Washrooms available?:</b> {park.hasWashrooms ? "yes" : "no"}
                             </div>
                             <div>
-                                Facilities available?: {park.hasFacilities ? "yes" : "no"}
+                                <b>Facilities available?:</b> {park.hasFacilities ? "yes" : "no"}
                             </div>
                             <div>
-                                Park advisories?: {park.hasAdvisories ? "yes" : "no"}
+                                <b>Park advisories?:</b> {park.hasAdvisories ? "yes" : "no"}
                             </div>
                             <div>
-                                Special Features?: {park.hasSpecialFeatures ? "yes" : "no"}
+                                <b>Special Features?:</b> {park.hasSpecialFeatures ? "yes" : "no"}
                             </div>
                             <div>
-                                Rating: {park.rating}
+                                <b>Rating:</b> {!!averageRating ? averageRating: "N/A"}
                             </div>
                             <div>
-                                Number of Favorites: {park.favoritesCount || 0}
+                                <b>Number of Favorites:</b> {park.favoritesCount || 0}
```

Comment:
"Number of users that like this park"

Reply:
I think the current is clear enough for intent